code/game/gamemodes/hivemind/hivemind.dm

```diff
@@ -52,6 +62,30 @@
 			H.hivemembers -= M
 			H.calc_size()
 
+/proc/handle_ejection(mob/living/carbon/human/H, datum/antagonist/hivemind/hive)
```

Comment:
again lol, make these procs on the antag datum or something what is wrong with you